app/views/admin/events/_all_with_comments.xlsx.axlsx

```diff
@@ -1,8 +1,20 @@
 wb.use_shared_strings = true
 wb.add_worksheet(name: 'events with comments') do |sheet|
-  bold_style = wb.styles.add_style(b: true )
+  bold_style = wb.styles.add_style( b: true )
```

Comment:
Space is _not_ needed inside parenthesis, a minor note, but I thought I would mention it since you are trying to keep everything in the right style :)